invenio_files_rest/helpers.py

```diff
@@ -166,11 +181,14 @@ def send_stream(stream, filename, size, mtime, mimetype=None, restricted=True,
     # Set cache-control
     if not restricted:
         rv.cache_control.public = True
-        cache_timeout = current_app.get_send_file_max_age(filename)
+        cache_timeout = current_app.get_send_file_max_age(filename) or max_age
         if cache_timeout is not None:
             rv.cache_control.max_age = cache_timeout
             rv.expires = int(time() + cache_timeout)
 
+    if no_cache:
```

Comment:
I would not add an extra param: if `restricted`, it should be `no-cache`. WDYT?
```suggestion
    else:
```

Reply:
could be, that would basically make it the default because unless restricted is changed it is `True`.